src/configProfileTemplate/config.ini

```diff
@@ -110,6 +110,8 @@ crypt_key=""
 # Override the real name on IRC connections
 #realname="{{realname}}"
 
+[notifications]
+enabled = true
```

Comment:
Could do with a comment explaining that a plugin is needed to actually send the notifications after enabling